city/schemas.py

```diff
@@ -0,0 +1,20 @@
+from pydantic import BaseModel, ConfigDict
+
+
+class CitySchema(BaseModel):
+    name: str
+    additional_info: str | None
+
+
+class CityCreateSchema(CitySchema):
+    pass
+
+
+class CityUpdateSchema(CitySchema):
+    pass
+
+
+class CityListSchema(CitySchema):
+    id: int
+
+    model_config = ConfigDict(from_orm=True)
```

Comment:
[change request] The from_orm method has been deprecated. Please use from_attributes instead.